setup.py

```diff
@@ -31,7 +31,6 @@
                       "bblfsh>=2.2.1,<3.0",
                       "modelforge>=0.5.4-alpha",
                       "sourced-engine>=0.5.1,<0.7",
-                      "pyspark>=2.2.0,<2.2.1",
```

Comment:
Should be `>=2.2.1,<2.3` if works

Reply:
but we have pyspark requirements from engine https://github.com/src-d/engine/blob/master/python/setup.py#L29 
should we fix anything in our setup?

Reply:
Right. I thought we removed this a long time, weird